src/Endpoints/DataTransferObjects/Organisation/OrganisationAttributes.php

```diff
@@ -15,8 +15,15 @@ class OrganisationAttributes extends DataTransferObject
 
     public ?string $plan;
 
+    public ?string $billing_mode;
+
     public ?string $billing_currency;
 
+    /**
+     * @var float|int|null
```

Comment:
The API returns integers if there are 0 decimals, and float if there are. To keep compatibility with PHP 7.4, the data type has to be specified this way in this case.